app/build.gradle

```diff
@@ -43,6 +43,16 @@ dependencies {
     compile 'com.android.support:cardview-v7:25.4.0'
     compile 'com.android.support.constraint:constraint-layout:1.0.2'
     compile 'me.dm7.barcodescanner:zxing:1.9.8'
+
+    androidTestCompile 'com.android.support:support-annotations:25.4.0'
+    androidTestCompile 'com.android.support.test:runner:0.5'
+    androidTestCompile 'com.android.support.test:rules:0.5'
```

Comment:
Would makes sense to extract the versions into a constant.

Reply:
👍  makes sense